### ISSUE TITLE
Make it possible to bind platform specific attributes

### ISSUE THREAD

**Issue by elena-p** [NativeScript/nativescript-angular#1665]
It seems that in NativeScript Angular project currently it is not possible to bind platform specific attributes (ios:/android:)

Here is a playground example: https://play.nativescript.org/?template=play-ng&id=BlyB9J

**Comment by replygirl** [comment]
i am very seriously interested in this

**Comment by edusperoni** [comment]
I don't think this is supported by angular. Can you give me a stackblitz example?

Binding `android:fontSize="15"` already works IIRC, but there's no way (angular limitation) to bind something like `android:[fontSize]=15`. I'm working on this exact thing at the moment and if you can make it work on angular I'll provide a way to do it in {N}+Angular.
